script.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const hamburger = document.querySelector('#hamburger_menu');
+    const menu = document.querySelector('.menu')
+    const close = document.querySelector('#close')
+    hamburger.onclick = function () {
+        hamburger.style.display = 'none';
+        menu.style.display = 'flex';
+        document.body.style.backgroundColor = 'hsl(236, 13%, 42%, 0.5)';
+    }
+    close.onclick = function () {
+        menu.style.display = 'none';
+        hamburger.style.display = 'block';
```

Comment:
It should be inside "if condition" handling the media query of max width 768 . as ...
      if (window.matchMedia("(max-width: 768px)").matches){
         hamburger.style.display = 'block';
       }  else
      {
         hamburger.style.display = 'none';
      }